### PR TITLE
Upload the language model

### PR DESCRIPTION
**Pull request by kuke**
Provide a language model for distribution.

### CODE REVIEW

File: deep_speech_2/README.md [hunk 1]
```diff
@@ -66,12 +66,36 @@ More help for arguments:
...
+
+```
+cd ./lm
+sh run.sh
+```
+
+After the downloading is completed, then
+
+```
+cd ..
```
Comment (xinghai-sun) [inline deep_speech_2/README.md]: Move L82 to Line77.

**Review by xinghai-sun — APPROVED** [review]

**Comment by pkuyym** [comment]
LGTM

**Review by pkuyym — APPROVED** [review]

File: deep_speech_2/README.md (hunk 1 above)
Comment (kuke) [reply deep_speech_2/README.md]: Done

**Review by kuke — COMMENTED** [review]
Follow the comment

**Pull request merged**